packages/glimmer-runtime/lib/vm/update.ts

```diff
@@ -1,7 +1,8 @@
 import { Scope, DynamicScope, Environment } from '../environment';
 import { Bounds, clear, move as moveBounds } from '../bounds';
 import { ElementStack, Tracker } from '../builder';
-import { LOGGER, Destroyable, Stack, LinkedList, InternedString, Dict, dict } from 'glimmer-util';
+import { LOGGER, Destroyable, Stack, FIXME, LinkedList, InternedString, Dict, dict } from 'glimmer-util';
```

Comment:
Some unused stuff here (`FIXME` is what my 👀 could spot, but tslint would tell you for sure)